### PR TITLE
Add Fatattr

### PR DESCRIPTION
**Pull request by pmiess**
add gnuwin32 package for fatattr

This package has some support with 33 packages reported on Repology

this is for 0install/0install.de-feeds#3

Moved from https://github.com/0install/repo.roscidus.com/pull/168

### CODE REVIEW

File: utils/fatattr.xml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0" ?>
+<?xml-stylesheet type='text/xsl' href='interface.xsl'?>
+<interface uri="https://apps.0install.net/utils/fatattr.xml" xmlns="http://zero-install.sourceforge.net/2004/injector/interface" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://zero-install.sourceforge.net/2004/injector/interface http://0install.de/schema/injector/interface/interface.xsd http://0install.de/schema/desktop-integration/capabilities http://0install.de/schema/desktop-integration/capabilities/capabilities.xsd">
+  <name>FatAttr</name>
+  <summary xml:lang="en">Fatattr: Display or change attributes on a filesystem</summary>
+  <description xml:lang="en">fatattr displays or changes attributes on an MS-DOS FAT
+       filesystem. If only a list of diles (or directories)
```
Comment (bastianeicher) [inline utils/fatattr.xml:7]: small typo: "diles" instead of "files"

File: utils/fatattr.xml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+  <entry-point binary-name="fatattr" command="run">
+    <needs-terminal/>
+    <name xml:lang="en">Fatattr</name>
+    <summary xml:lang="en">fatattr: Display or change attributes on a FAT filesystem</summary>
+    <description xml:lang="en">fatattr displays or changes attributes on an MS-DOS FAT
+       filesystem. If only a list of diles (or directories)
+       are given, it displays the attributes for those files or
+       directories. If given a list of attributes, it will add
+       (+) or remove (-) the specified attributes from the
+       given list of files or directories.</description>
```
Comment (bastianeicher) [inline utils/fatattr.xml:31]: When the `<summary>` and `<description>` of an `<entry-point>` are the same as the top-level equivalents in the feed, it's usually better to just leave them out to keep the file shorter.

File: utils/fatattr.xml (hunk 1 above)
Comment (pmiess) [reply utils/fatattr.xml]: fixed

File: utils/fatattr.xml (hunk 2 above)
Comment (pmiess) [reply utils/fatattr.xml]: deleted

**Pull request merged**